src/alpha-lab/dense-pose.md

```diff
@@ -1,78 +1,145 @@
+Have you ever wondered which body parts we gaze upon while conversing with others? Where a professional basketball player looks just before passing? Does hand movement play a role when delivering a speech?
 
-If you ever asked yourself any of these questions, but you always hesitate to analyse the eye-tracking data, because you didn't want to manually label frames, well, we have something for you, keep reading! 
+Here we introduce a powerful and time-saving automation that can help answer these questions. It works by identifying body parts of people in the scene video and recording whether they were gazed at!
 
-With the introduction of [Face Mapper Enrichment](/invisible/explainers/enrichments/face-mapper), we gave you the power to address how much and which faces are looked at in an environment. Now, we are giving you a way to determine which part of a body is being gazed. With the help of *DensePose* (part of [detectron2](https://ai.facebook.com/tools/detectron2/), Meta AI), you will be able to automatically identify, and segment body parts on a video and check if they have been gazed.
+::: tip
+<b>This sounds really complicated...</b><br>
+Don't worry! We've made this powerful tool easy-to-use, so you can get started right away – no coding required! By the end, you'll have a video like the one above, and a CSV file documenting which body parts were gazed.
+:::
 
+## Powered by DensePose
 <details>
-    <summary>Do you want to know more about <b>DensePose</b>? Click here</summary>
+    <summary>This tool employs <b>DensePose</b> (<a href="https://ai.facebook.com/tools/detectron2/">detectron2</a>, Meta AI). Click here to learn more</summary><br>
 <!-- This is collapsed   -->
-<h3>What is <b>DensePose</b>?</h3><br>
-    DensePose is a method for dense human pose estimation and dense human body part segmentation. It is based on the Mask R-CNN architecture and is trained on the COCO dataset. DensePose is now a part of Detectron2's framework. You can read all the details in their paper <a href="https://arxiv.org/abs/1802.00434">DensePose: Dense Human Pose Estimation In The Wild.</a>
+    DensePose is a method for dense human pose estimation and dense human body part segmentation. It's based on the Mask R-CNN architecture and is trained on the COCO dataset. DensePose is now a part of Detectron2's framework. You can read all the details in their paper <a href="https://arxiv.org/abs/1802.00434">DensePose: Dense Human Pose Estimation In The Wild.</a>
 </details>
 <br>
 
+## What you'll need
+
+1. A **[Raw data export]()** from Pupil Cloud. The downloaded folder contains one subfolder for every recording in the project. Each subfolder contains a scene video, gaze.csv, and other files. We will work with those subfolders, so have them at hand
```

Comment:
Link is missing. Might also make sense to use the name as it is displayed in the Cloud Drive download options?

Reply:
Added! Good catch! as of now is the raw enrichment download.